mesop/components/markdown/markdown.ts

```diff
@@ -1,4 +1,14 @@
-import {ChangeDetectorRef, Component, Input} from '@angular/core';
+import {
+  ChangeDetectorRef,
+  Component,
+  Input,
+  ElementRef,
+  Renderer2,
+  SecurityContext,
+} from '@angular/core';
+import {DomSanitizer, SafeHtml} from '@angular/platform-browser';
+import {marked} from 'marked';
```

Comment:
i think you need this shim: https://github.com/google/mesop/commit/8a4f375d56c498e95a47bc9ef8282d7b7fa39f69#diff-0bfb67474f7fa139128300039a775d666a921f24b2dee795d8d725cb2299300b to make it work downstream

Reply:
Ah thanks. I remember you mentioning this before. But was not able to find it. Updated.